src/diffusers/pipelines/kandinsky2_2/pipeline_kandinsky2_2_prior_emb2emb.py

```diff
@@ -458,8 +462,11 @@ def __call__(
                 Conceptually, indicates how much to transform the reference `emb`. Must be between 0 and 1. `image`
                 will be used as a starting point, adding more noise to it the larger the `strength`. The number of
                 denoising steps depends on the amount of noise initially added.
-            emb (`torch.FloatTensor`):
-                The image embedding.
+            image (`PIL.Image.Image` or `torch.FloatTensor`):
+                The image embedding that will be used as the starting point for the prior diffusion process. Can accpet
+                image or image embedding. If passing an image as `image`, either as a PIL.Image.Image, or a pytorch
+                tensor represent the image batch, it will first be encoded into image embedding using image_encoder; If
+                passing image embedding directly, it will not be encoded again.
```

Comment:
```suggestion
                The image embedding that will be used as the starting point for the prior diffusion process. Can accept
                image or image embedding. If passing an image as `image`, either as a PIL.Image.Image, or a PyTorch
                tensor, it will first be encoded into image embedding using `image_encoder`. If
                passing image embedding directly, it will not be encoded again.
```